docs/source/changes.md

```diff
@@ -120,3 +120,7 @@ authors:
 - `epochs_tmin` and `epochs_tmax` were named incorrectly in some test config
   files.
   ({{ gh(340) }} by {{ authors.crsegerie }})
+- We now reject bad epochs trespassing `ica_reject` before plotting the 
+  figures in the `proc-ica_report`.
```

Comment:
Haha, nicely phrased, I giggled :)

But I'm not sure if it's easy to understand for someone who wasn't involved in our discussion to understand what has been fixed here. Proposal:

```suggestion
- When creating the "overlay" plot to show the effects of ICA cleaning, we
  previously plotted the data without applying the pre-ICA peak-to-peak
  rejection thresholds, producing misleading figures in some situations. This
  has now been fixed.
```

Reply:
I see you've meanwhile updated the changelog, so ignore this comment